src/dispose.ts

```diff
@@ -0,0 +1,37 @@
+import * as vscode from 'vscode';
+
```

Comment:
Always update related issues, or reference issue using `#` in commit message.
For now it is not clear how much of this actually works.
From what I see. The highlight editing and undo/redo is not fully implemented.